nautobot/docs/administration/nautobot-server.md

```diff
@@ -0,0 +1,476 @@
+
+`--no-clearsessions`<br>
+Do not automatically clean out expired sessions.
+
+`--no-collectstatic`<br>
+Do not automatically collect static files into a single location.
+
+`--no-invalidate-all`<br>
+Do not automatically invalidate cache for entire application.
+
+`--no-migrate`<br>
+Do not automatically perform any database migrations.
+
+`--no-remove-stale-contenttypes`<br>
+Do not automatically remove stale content types.
+
+`--no-trace-paths`<br>
+Do not automatically generate missing cable paths.
+
+```no-highlight
+$ nautobot-server post_upgrade
+Performing database migrations...
+Operations to perform:
+  Apply all migrations: admin, auth, circuits, contenttypes, dcim, extras, ipam, sessions, taggit, tenancy, users, virtualization
+Running migrations:
+  No migrations to apply.
+
+Generating cable paths...
+Found no missing circuit termination paths; skipping
+Found no missing console port paths; skipping
+Found no missing console server port paths; skipping
+Found no missing interface paths; skipping
+Found no missing power feed paths; skipping
+Found no missing power outlet paths; skipping
+Found no missing power port paths; skipping
+Finished.
+
+Collecting static files...
+
+0 static files copied to '/opt/nautobot/static', 965 unmodified.
+
+Removing stale content types...
+
+Removing expired sessions...
+
+Invalidating cache...
+```
+
+### `renaturalize`
+
+`nautobot-server renaturalize [app_label.ModelName [app_label.ModelName ...]]`
+
+Recalculate natural ordering values for the specified models. 
+
+This defaults to recalculating natural ordering on all models which have one or more fields of type `NaturalOrderingField`:
+
+```no-highlight
+$ nautobot-server renaturalize
+Renaturalizing 21 models.
+dcim.ConsolePort.name (_name)... 196
+dcim.ConsoleServerPort.name (_name)... 0
+dcim.PowerPort.name (_name)... 392
+dcim.PowerOutlet.name (_name)... 0
+dcim.Interface.name (_name)... 7161
+dcim.FrontPort.name (_name)... 0
+dcim.RearPort.name (_name)... 0
+dcim.DeviceBay.name (_name)... 0
+dcim.InventoryItem.name (_name)... 1
+dcim.Device.name (_name)... 208
+dcim.ConsolePortTemplate.name (_name)... 2
+dcim.ConsoleServerPortTemplate.name (_name)... 0
+dcim.PowerPortTemplate.name (_name)... 4
+dcim.PowerOutletTemplate.name (_name)... 0
+dcim.InterfaceTemplate.name (_name)... 221
+dcim.FrontPortTemplate.name (_name)... 0
+dcim.RearPortTemplate.name (_name)... 0
+dcim.DeviceBayTemplate.name (_name)... 0
+dcim.Rack.name (_name)... 156
+dcim.Site.name (_name)... 22
+virtualization.VMInterface.name (_name)... 0
+Done.
+```
+
+You may optionally specify or more specific models (each prefixed with its app_label) to renaturalize:
+
+```no-highlight
+$ nautobot-server renaturalize dcim.Device
+Renaturalizing 1 models.
+dcim.Device.name (_name)... 208
+Done.
+```
+
+### `runjob`
+
+`nautobot-server runjob [job]`
+
+Run a job (script, report) to validate or update data in Nautobot.
```

Comment:
perhaps (formally script or report)?

I am personally less opinionated about how users intend to use jobs, but lack a replacement of `to validate or update data in Nautobot`

Reply:
This was just lifted from the help text for `run job`. If we update it here, we should update both: 

```
$ nautobot-server runjob -h
usage: nautobot-server runjob [-h] [--commit] [--version] [-v {0,1,2,3}] [--settings SETTINGS]
                              [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--force-color]
                              [--skip-checks]
                              job

Run a job (script, report) to validate or update data in Nautobot
...
```